### ISSUE TITLE
Issue with conditional in Makefile

### ISSUE THREAD

**Issue by tghosth** [OWASP/ASVS#1678]
Hi @ike,

I was experimenting with running `make docker` and then `make all` separately.

My understanding is that `make docker` will just create the docker image and load it into the local docker image.

Looking at the following code...

```
.PHONY: docker
docker:
	if ! docker image inspect asvs/documentbuilder:latest --format="checking docker image" &> /dev/null; then \
		docker build --build-arg CERT_FILE=docker/cert --tag asvs/documentbuilder:latest --network host docker; \
	fi;
```

...my understanding is that if I have already run `make docker` and then I run `make all`, then this if statement will get run again but since the image now exists, the `docker build` command should not get run.

However, it looks to me like the `docker build` command does in fact get run regardless of the output of `docker image inspect`.

Are you able to fix this?

**Comment by tghosth** [comment]
This should be fixed before #1638

**Comment by ike** [comment]
@tghosth yes, I believe the appropriate solution is to put the image on a repository and only pull in the action, instead of building in the action at all.

We could potentially have an action that built and pushed the image if the `docker` dir changes.

**Comment by tghosth** [comment]
happy for however you want to address it but it would be good to get it sorted

**Comment by ike** [comment]
#1680 removes the `docker` target from any of the other targets, so this issue should be resolved now. You can still build the docker image using `make docker`.

**Comment by tghosth** [comment]
Ok so this should be resolved as part of issue #1638

**Comment by ike** [comment]
Fixed with https://github.com/OWASP/ASVS/pull/1680
